traffic_portal/test/integration/specs/Profiles.spec.ts

```diff
@@ -94,12 +94,9 @@ profiles.tests.forEach(async profilesData => {
         });
     });
 });
-
 describe('Clean up API for Profiles', () => {
-    afterAll(async function () {
-        it('Cleanup', async () => {
-            await api.UseAPI(profiles.cleanup);
-        });
-    })
+    it('Cleanup', async () => {
+        await api.UseAPI(profiles.cleanup);
+    });
```

Comment:
Was this file supposed to be changed?

Reply:
After couple test and research. The afterAll can only be use one in a spec file regardless how many describe you put it in. So with this afterAll, the clean up API will not work.

Reply:
> After couple test and research. The afterAll can only be use one in a spec file regardless how many describe you put it in.

Where did you hear that? [The Jasmine `afterAll` docs](https://jasmine.github.io/api/3.9/global.html#afterAll) say:

> Run some shared teardown once after all of the specs in the `describe` are run.

So you should be able to have one per `describe`. After all, hasn't this been working for a while now?

Reply:
@ocket8888 I tested it. You can look at the old test packaged that the clean up hasn't been run for a while. Only 1 afterAll run per spec doesn't matter you put the second afterAll in another describe or outside other describe. The test working for a while because the cleanup for the profiles did not touch other test.

Reply:
You can check it in this PR for example: https://github.com/apache/trafficcontrol/pull/6233/checks?check_run_id=3676058657

Reply:
Yeah, I wouldn't expect it to actually run. Like the docs say:

> Run some shared teardown once after all of the specs in the `describe` are run.

That `describe` contains no specs. The `afterAll` should be in the same `describe` as the specs _after all_ of which the `afterAll` block will run.

See in this demo:

https://jsfiddle.net/62pefumq/

After running it, there should be two debug-level logs in the console: "in afterAll 1" and "in afterAll 3". "in afterAll 2" doesn't get printed because its `describe` contains no specs.

Reply:
@ocket8888 Ok, got you. I fixed it.